unesp-dracena.csl

```diff
@@ -0,0 +1,589 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR">
+  <info>
+    <title>UNESP - Campus de Dracena - ABNT - Autores abreviados (Portuguese - Brazil)</title>
+    <title-short>UNESP-FCAT</title-short>
+    <id>http://www.zotero.org/styles/unesp-dracena</id>
+    <link href="http://www.zotero.org/styles/unesp-dracena" rel="self"/>
+    <link href="http://www.zotero.org/styles/unesp-dracena.csl" rel="template"/>
+    <link href="http://www.dracena.unesp.br/#!/servicos/biblioteca/estilo-para-zotero/" rel="documentation"/>
+    <author>
+      <name>Fabio Sampaio Rosas </name>
+      <email>fabio.rosas@unesp.br</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>The Brazilian Standard Style in accordance with ABNT-NBR 10520.2002 and ABNT-NBR 6023.2002</summary>
+    <updated>2018-06-08T11:25:37+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="pt-BR">
+    <terms>
+      <term name="month-01" form="short">jan.</term>
+      <term name="month-02" form="short">fev.</term>
+      <term name="month-03" form="short">mar.</term>
+      <term name="month-04" form="short">abr.</term>
+      <term name="month-05" form="short">maio</term>
+      <term name="month-06" form="short">jun.</term>
+      <term name="month-07" form="short">jul.</term>
+      <term name="month-08" form="short">ago.</term>
+      <term name="month-09" form="short">set.</term>
+      <term name="month-10" form="short">out.</term>
+      <term name="month-11" form="short">nov.</term>
+      <term name="month-12" form="short">dez.</term>
```

Comment:
All these month locales are already set like that via the language setting. You'd only set these if you'd wanted to override them. See here for the pt-BR locale file: https://github.com/citation-style-language/locales/blob/master/locales-pt-BR.xml